node/src/chain_spec.rs

```diff
@@ -564,6 +580,85 @@ fn amplitude_genesis(
 			members: signatories.clone(),
 			..Default::default()
 		},
+		tokens: amplitude_runtime::TokensConfig {
+			// Configure the initial token supply for the native currency and USDC asset
+			balances: token_balances,
+		},
+		issue: amplitude_runtime::IssueConfig {
+			issue_period: amplitude_runtime::DAYS,
+			issue_minimum_transfer_amount: 1000,
+			limit_volume_amount: None,
+			limit_volume_currency_id: XCM(DOT),
+			current_volume_amount: 0u32.into(),
+			interval_length: (60u32 * 60 * 24).into(),
+			last_interval_index: 0u32.into(),
+		},
+		redeem: amplitude_runtime::RedeemConfig {
+			redeem_period: foucoco_runtime::DAYS,
+			redeem_minimum_transfer_amount: 1000,
+			limit_volume_amount: None,
+			limit_volume_currency_id: XCM(DOT),
+			current_volume_amount: 0u32.into(),
+			interval_length: (60u32 * 60 * 24).into(),
+			last_interval_index: 0u32.into(),
+		},
+		replace: amplitude_runtime::ReplaceConfig {
+			replace_period: foucoco_runtime::DAYS,
+			replace_minimum_transfer_amount: 1000,
+		},
+		security: amplitude_runtime::SecurityConfig {
+			initial_status: if start_shutdown {
+				amplitude_runtime::StatusCode::Shutdown
+			} else {
+				amplitude_runtime::StatusCode::Error
+			},
+		},
+		oracle: amplitude_runtime::OracleConfig {
+			max_delay: u32::MAX,
+			oracle_keys: vec![
+				Key::ExchangeRate(XCM(ForeignCurrencyId::DOT)),
+				Key::ExchangeRate(amplitude_runtime::WRAPPED_CURRENCY_ID),
```

Comment:
Can we change this to 
```
				Key::ExchangeRate(amplitude_runtime::DefaultWrappedCurrencyId::get()),

```
instead? Boils down to the same thing but I prefer the other function because it mentions that it's the default.

Reply:
done